package.json

```diff
@@ -2,10 +2,8 @@
   "name": "red-crescent-app",
   "version": "0.1.0",
   "private": true,
-  "homepage": "http://mohammedi-haroune.github.io/red-crescent-webapp",
   "dependencies": {
     "@material-ui/core": "^4.6.1",
-    "gh-pages": "^2.1.1",
```

Comment:
why removing gh-pages ?

Reply:
Sorry my mistake, i didn't **git pull** when i created the hospitals-page branch, so i didn't have the version of package.json where it has gh-pages. 
now it's fixed in 098b66e